xterm/index.cgi

```diff
@@ -181,6 +181,18 @@ my $user = $access{'user'};
 if ($user eq "*") {
 	$user = $remote_user;
 	}
+
+# Switch to given user
+if (&webmin_user_is_admin()) {
```

Comment:
Instead just check if `$user eq "root"`

Reply:
No, no .. why? I would prefer to use a specialized sub for it.

Reply:
In this specific case, if the user can access the shell as `root`, then have the power to run as any user. Checking `webmin_user_is_admin` may have a different meaning.

Reply:
Got it. Fixed.